Fix typos in lib/public subdirectory

luzpaz opened the pull request:
Found via `codespell -q 3 -S l10n -L jus ./lib/private`

Signed-off-by: luz paz <luzpaz@github.com>

PVince81 reviewed (approved):
:+1: looks fine

come-nc commented on lib/public/Accounts/IAccount.php:
```diff
@@ -135,7 +135,7 @@ public function getAllProperties(): Generator;
 
 	/**
-	 * Returns the requestes propery collection (multi-value properties)
+	 * Returns the requests property collection (multi-value properties)
```
```suggestion
	 * Returns the requested property collection (multi-value properties)
```

luzpaz commented:
Squashed. Shall I also rebase or not necessary?

come-nc commented:
> Squashed. Shall I also rebase or not necessary?

I do not think a rebase is needed, but the DCO bot is somehow unhappy, can you look into it?

luzpaz commented:
Not sure why.

szaimen commented:
I think it expects a different mail but this is not a problem and we can just set it to skip, afaics.

skjnldsv commented:
Please fix conflict and we can merge

szaimen committed:
Fix typos in lib/public subdirectory

Found via `codespell -q 3 -S l10n -L jus ./lib/public`

Signed-off-by: luz paz <luzpaz@github.com>

Update lib/public/Accounts/IAccount.php

Signed-off-by: luz paz <luzpaz@github.com>

Signed-off-by: Simon L <szaimen@e.mail.de>
Co-Authored-By: Côme Chilliet <91878298+come-nc@users.noreply.github.com>
